app/facades/foods_index_search_result_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodsIndexSearchResultFacade
+
+    def self.service(search) 
```

Comment:
This is nit, more sharing for context/info: Typically, facade method names are named after the data object they are providing. So it's more often seen that this would be called `food_results` or something like that